.git-blame-ignore-revs

```diff
@@ -0,0 +1,11 @@
+# This file contains the list of commits to exclude from 'git blame'.
+# Such commits do not meaningfully contribute to git history, and include
+# large-scale mechanical changes like code formatting style changes.
+#
+# To set this file as the default ignore file for 'git blame', run:
+# ```shell
+#   git config blame.ignoreRevsFile .git-blame-ignore-revs
+# ```
+
+# Refresh clang-format
+2844576efc9b81f1900f567732f9e67526f1d3f6
```

Comment:
I'll have to send this as a follow up so that I can get the commit hash right after squashing (easier to keep the comment updates and the mechanical stuff separate).